schematic/models/GE_Helpers.py

```diff
@@ -151,30 +151,17 @@ def add_expectation_suite_if_not_exists(self) -> ExpectationSuite:
         Returns:
             saves expectation suite and identifier to self
         """
-        self.expectation_suite_name = "Manifest_test_suite"
-        # Get a list of all expectation suites
-        suite_names = self.context.list_expectation_suite_names()
-        # Get a list of all checkpoints
-        all_checkpoints = self.context.list_checkpoints()
-
-        # if the suite exists, delete it
-        if self.expectation_suite_name in suite_names:
-            self.context.delete_expectation_suite(self.expectation_suite_name)
-
-            # also delete all the checkpoints associated with the suite
-            if all_checkpoints:
-                for checkpoint_name in all_checkpoints:
-                    self.context.delete_checkpoint(checkpoint_name)
-
-        self.suite = self.context.add_expectation_suite(
+        self.expectation_suite_name = f"Manifest_test_suite_{uuid.uuid4()}"
+        expectation_suite = self.context.add_expectation_suite(
             expectation_suite_name=self.expectation_suite_name,
         )
+        self.suite = expectation_suite
```

Comment:
I found issues when tests were running concurrently where great expectation test suites were stepping over eachother, causing tests to break in Un-expected places. With this change the JSON file that is created for each test suite run is unique, and cleaned up after the test suite has ran.

No more will another test clean these up.

Reply:
The same notes about concurrency also apply when this code is running as an API server. If 2 users were sending in a request at the same time this is likely to have caused issues if they lined up just perfectly.